docs/source/conf.py

```diff
@@ -66,6 +66,24 @@ def setup(app):
 #
 html_theme = 'sphinx_rtd_theme'
 
+html_static_path = ['../static']
+
+html_css_files = [
+	# override wide tables in RTD theme
```

Comment:
Just copy-paste the original comment

Reply:
"override wide tables in RTD theme" is the original comment **here**. you mean change it to "save the table width" ?

Reply:
both of them for full information